.github/workflows/tests.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/rauc/rauc/rauc-ci:latest
+      image: ghcr.io/jluebbe/rauc/rauc-ci:latest
```

Comment:
This doesn't match the commit title (which still includes `WIP` anyway).

Reply:
l'll fix the commit message.

Reply:
Yes, see the last paragraph in the description. :)

Reply:
Got it, thx!